lib/node_modules/@stdlib/blas/ext/base/ssort2ins/lib/ndarray.native.js

```diff
@@ -63,13 +64,14 @@ function ssort2ins( N, order, x, strideX, offsetX, y, strideY, offsetY ) {
 		flg *= -1.0; // reversing order
 		order *= -1.0;
 		strideX *= -1;
```

Comment:
I believe you need to remove this line. L69 is assuming the original `strideX`.

Reply:
Hmm...this implementation is a bit trickier.

Reply:
I'd double check that the behavior you've introduced here matches the behavior of the previous implementation. In short, what is provided to `addon` to be the same before and after the changes introduced in this PR. And I'd confirm this for positive and negative strides.

Reply:
So are there any changes required? @kgryte

Reply:
I'm not sure, which is why I encourage you to check. The implementation as is looks suspect, but I don't have the bandwidth atm to confirm.